packages/apps/storybook/src/create-imodel/CreateIModel.css

```diff
@@ -0,0 +1,16 @@
+/*---------------------------------------------------------------------------------------------
+ * Copyright (c) Bentley Systems, Incorporated. All rights reserved.
+ * See LICENSE.md in the project root for license terms and full copyright notice.
+ *--------------------------------------------------------------------------------------------*/
+.iModelCustomized > .iac-themeProvider {
+    display: flex;
+    justify-content: center;
+}
+
+.iModelCustomized .container {
```

Comment:
Are these the fixes for thumbnail going outside of file uplaod container? I expected these to be in package, not application.. I will run it tomorrow to check :/

Reply:
Nope, these are just to fix the customized IModel which we created ourself in storybook, the thumnail changes are in package itself

Reply:
right, but image still goes out of borders...
<img width="775" alt="image" src="https://github.com/iTwin/admin-components-react/assets/81580355/2b68cf1f-5d60-476e-9c50-579ff5d6f8e5">

Reply:
Yes, i did.. Inspect the container of your image
<img width="279" alt="image" src="https://github.com/iTwin/admin-components-react/assets/81580355/aceed5af-c01e-4399-9ea3-ec6e13df79c6">

it has hardcoded height, which is bigger than file upload..
You did not change that.. You have just removed the border (why?)

Reply:
done + added border back, not really sure why I removed it maybe the migration tool did something